Api/RequestHandler.php

```diff
@@ -94,6 +94,10 @@ protected function buildBaseParams( $payment, $order, $apiToken )
             $baseParams['cvvResultCode'] = $payment->getCcCidStatus();
         }
 
+        if ($payment->getMethod() === 'chcybersource' && $payment->getAdditionalInformation('auth_avs_code')) {
```

Comment:
The method buildParamsAdditionalInfo() is built to handle gateway specific data that is set onto the payme'ts additional information. This needs to be moved to that method.